src/Workspaces/Remote/Core/RemoteHostAssetSerialization.cs

```diff
@@ -31,27 +31,28 @@ public static async Task WriteDataAsync(ObjectWriter writer, IRemotableDataServi
             if (checksums.Length == 1)
             {
                 var checksum = checksums[0];
-                var remotableData = (await remotableDataService.GetRemotableDataAsync(scopeId, checksum, cancellationToken).ConfigureAwait(false)) ?? RemotableData.Null;
+
+                var asset = (await assetStorage.GetAssetAsync(scopeId, checksum, cancellationToken).ConfigureAwait(false)) ?? SolutionAsset.Null;
```

Comment:
1. should GetAssetAsync return SolutionAsset.Null if it can't find the item?
2. when does GetAssetAsync return null?  seems... wrong that it could do that as that would mean we're not able to sync assets right?

Reply:
regardless, could use docs here.

Reply:
1. I don't think so. `SolutionAsset.Null` represents `null` literal in serialized data. 
2. Yeah, seems like we will null-ref if it does :(. Will fix that.